src/contracts/exchange/NiftyswapExchange20.sol

```diff
@@ -0,0 +1,812 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.7.4;
+pragma experimental ABIEncoderV2;
+import "../interfaces/INiftyswapExchange20.sol";
+import "../utils/ReentrancyGuard.sol";
+import "@0xsequence/erc-1155/contracts/interfaces/IERC20.sol";
+import "@0xsequence/erc-1155/contracts/interfaces/IERC165.sol";
+import "@0xsequence/erc-1155/contracts/interfaces/IERC1155.sol";
+import "@0xsequence/erc-1155/contracts/interfaces/IERC1155TokenReceiver.sol";
+import "@0xsequence/erc-1155/contracts/tokens/ERC1155/ERC1155MintBurn.sol";
+
+/**
+ * This Uniswap-like implementation supports ERC-1155 standard tokens
+ * with an ERC-20 based token used as a currency instead of Ether.
+ *
+ * See https://github.com/0xsequence/erc20-meta-token for a generalized
+ * ERC-20 => ERC-1155 token wrapper
```

Comment:
You can delete these two lines since we don't need to wrap ERC-20 into ERC-1155 with this contract.